package.json

```diff
@@ -16,15 +16,15 @@
     "ctest": "cross-env BROWSER=chromium node test/test.js",
     "ftest": "cross-env BROWSER=firefox node test/test.js",
     "wtest": "cross-env BROWSER=webkit node test/test.js",
-    "test": "cross-env BROWSER=all node test/test.js",
+    "test": "run-s -c ctest ftest wtest",
```

Comment:
it was nice to have a single `test` command for the following usecase.

Consider I'm debugging a single test across 3 browsers. I could just focus it and do `npm test` to run 3 browsers and get a single report. Whereas in this case, I'll get three reports (which was not convenient).